### PR TITLE
Mlc 410 drift to plotly

### PR DESCRIPTION
**Pull request by benisraeldan**

### CODE REVIEW

File: deepchecks/checks/distribution/dist_utils.py [hunk 1]
```diff
@@ -25,29 +26,49 @@
...
+                marker=dict(
+                    color=color,
+                    line=dict(color='rgb(248, 248, 249)', width=1)
+                ),
+                offsetgroup=0,
+                base=range_tuple[0],
+                showlegend=False
+
+            ))
+        if drift_score >= range_tuple[1]:
```
Comment (JKL98ISR) [inline deepchecks/checks/distribution/dist_utils.py:58]: can use min/max instead of the if

**Review by JKL98ISR — CHANGES_REQUESTED** [review]

File: deepchecks/checks/distribution/dist_utils.py [hunk 2]
```diff
@@ -25,29 +26,49 @@
...
-            axes.barh(0, range_tuple[1] - range_tuple[0], left=range_tuple[0], color=color)
-    axes.set_title('Drift Score - ' + drift_type)
-    axes.set_xlim([0, stop])
-    axes.set_yticklabels([])
+            bars.append(go.Bar(
+                x=[drift_score - range_tuple[0]], y=['Drift Score'],
+                orientation='h',
+                marker=dict(
+                    color=color,
+                    line=dict(color='rgb(248, 248, 249)', width=1)
```
Comment (JKL98ISR) [inline deepchecks/checks/distribution/dist_utils.py:51]: also not sure of choosing color is really needed, plotly has nice colors

**Comment by JKL98ISR** [comment]
also not sure where it is but psi bars have no legend so you don't see who's test and who's train

File: deepchecks/checks/distribution/plot.py [hunk 3]
```diff
@@ -37,3 +43,126 @@ def plot_density(data, xs, color='b', alpha: float = 0.7, **kwargs) -> np.ndarra
...
+        )
+
+        test_bar = go.Bar(
+            x=cat_df.index, y=cat_df['Test dataset'],
+            orientation='v',
+            marker=dict(
+                color=colors['Test'],
+            ),
+            name='Test Dataset',
+            showlegend=True
```
Comment (JKL98ISR) [inline deepchecks/checks/distribution/plot.py:144]: isn't it on default? also orientation v?

**Review by JKL98ISR — APPROVED** [review]

**Review by noamzbr — APPROVED** [review]

**Pull request merged**